### PR TITLE
adding some branding styles

### PR DESCRIPTION
**Pull request by stevenplewis**

### CODE REVIEW

**Comment by chadwhitacre** [comment]
Woo-hoo! Thanks @stevenplewis! 😁 💃

File: src/pages/index.astro [hunk 1]
```diff
@@ -30,26 +35,32 @@ import Button from "../components/Button.astro";
...
+      <div class="highlight-box">
+        <h3>Step Two</h3>
+        <p>Self-report annually.</p>
+        <ol>
+          <li>number of full-time employed developers</li>
+          <li>payments to maintainers by platform or foundation</li>
+        </ol>
+      </div>
+      <div class="footer-cta-box">
+        <h2>Let's make it happen.</h2>
```
Comment (vladh) [inline src/pages/index.astro:58]: Optional idea: if we give this `font-weight: 1.8rem`, the text aligns with the button which is pleasant.

File: src/components/Prose.astro [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <article
-  class="prose prose-app my-12 max-w-none text-base prose-strong:text-white prose-table:text-base prose-a:text-app-purple hover:prose-a:text-app-pink prose-headings:text-app-green text-white"
+  class="prose prose-app my-12 max-w-none text-base prose-table:text-base prose-a:text-app-green hover:prose-a:text-app-pink text-white"
```
Comment (vladh) [inline src/components/Prose.astro:2]: This change broke the text on the member pages, maybe there's a quick fix here?

File: src/pages/index.astro [hunk 3]
```diff
@@ -59,4 +70,55 @@ import Button from "../components/Button.astro";
...
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    color: #fff;
+    font-weight: 600;
+  }
+
+  h1.hero-header {
```
Comment (vladh) [inline src/pages/index.astro:83]: Not really sure why, but the font sizes look a bit disproportionate on mobile, the hero text is very big and hits the right border, and the body text feels small. Here it is on iPhone SE dimensions:

**Review by vladh — CHANGES_REQUESTED** [review]
Love it! Thank you! Just a few things I noticed.

**Comment by vladh** [comment]
@stevenplewis Let me know if you'd like to address the above, if so I would recommend a revert of the merge, otherwise I can also have a look!

File: src/pages/index.astro (hunk 1 above)
Comment (stevenplewis) [reply src/pages/index.astro:58]: ah, but it also creates a widow at default size. I don't think it's worth it.

File: src/components/Prose.astro (hunk 2 above)
Comment (stevenplewis) [reply src/components/Prose.astro:2]: ah, yeah, what you said makes sense.

**Comment by chadwhitacre** [comment]
> a revert of the merge

Probably easiest to just have @stevenplewis do a follow-up PR at this point? We don't really have traffic on the site yet.

https://plausible.io/osspledge.com

**Comment by vladh** [comment]
Sure thing!

**Pull request merged**